packages/core/admin/admin/src/content-manager/pages/App/index.js

```diff
@@ -16,7 +16,7 @@ import getTrad from '../../utils/getTrad';
 import DragLayer from '../../components/DragLayer';
 import ModelsContext from '../../contexts/ModelsContext';
 import CollectionTypeRecursivePath from '../CollectionTypeRecursivePath';
-import ComponentSettingsView from '../ComponentSetttingsView';
+import ComponentSettingsView from '../ComponentSettingsView';
```

Comment:
I've tried to rename all of the things like this. I have tooling for it, but it's an add-on, so I can easily fail to run it. I generally hope that CI will catch me when I mess up. That said, see elsewhere...